ui/selectors/selectors.js

```diff
@@ -1757,6 +1762,25 @@ export function getCustomTokenAmount(state) {
   return state.appState.customTokenAmount;
 }
 
+export function getUpdatedAndSortedAccounts(state) {
```

Comment:
I think this function should have unit tests?

Reply:
Yes, it would be nice to have a unit test for this selector to verify that the order is expected,
```ts
  it('#getUpdatedAndSortedAccounts', () => {
    const pinnedAccountState = {
      ...mockState,
      metamask: {
        ...mockState.metamask,
        pinnedAccountList: ['0x2...', '0x0...', '0x1...'],
      },
    };
    const expectedResult = [
      '0x2...',
      '0x0...',
      '0x1...',
      mockState.metamask.accounts,
    ];
    expect(
      selectors
        .getUpdatedAndSortedAccounts(pinnedAccountState)
        .toStrictEqual(expectedResult),
    );
  });
  ```